api/main_endpoints/routes/AuditLog.js

```diff
@@ -0,0 +1,46 @@
+const express = require('express');
+const router = express.Router();
+const AuditLog = require('../models/AuditLog');
+const { OK, BAD_REQUEST, UNAUTHORIZED } = require('../../util/constants').STATUS_CODES;
+
+const { decodeToken, checkIfTokenSent } = require('../util/token-functions.js');
+
+const logger = require('../../util/logger');
+
+router.get('/getAuditLogs', async (req, res) => {
+  if (!checkIfTokenSent(req)) {
+    logger.warn('/getAuditLogs was requested without a token');
+    return res.sendStatus(UNAUTHORIZED);
+  }
+
+  const decodedPayload = await decodeToken(req);
+
+  if (!decodedPayload) {
+    logger.warn('/getAuditLogs was requested with an invalid token');
+    return res.sendStatus(UNAUTHORIZED);
+  }
+
+  if (decodedPayload.accessLevel < 2) {
+    return res.sendStatus(UNAUTHORIZED);
+  }
```

Comment:
i think you can use `checkIfTokenValid` to verify that they are an admin, see

https://github.com/SCE-Development/Clark/blob/7fead735a77d6a97d5b6254bec74b787bbee1511/api/main_endpoints/util/token-functions.js#L68